### PR TITLE
Guideline editor

### PR DESCRIPTION
**Pull request by AlFasGD**
Noah you are FUCKED

### CODE REVIEW

**Comment by Altenhh** [comment]
OH LAWD PLEASE HELP

File: GDAPI/GDAPI/Utilities/Objects/GeometryDash/General/GuidelineColor.cs [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    public struct GuidelineColor : IComparable<GuidelineColor>
+    {
+        /// <summary>Represents the value of the orange color in the guideline.</summary>
+        public const float TransparentValue = 0.7f;
+        /// <summary>Represents the value of the yellow color in the guideline.</summary>
+        public const float OrangeValue = 0.8f;
+        /// <summary>Represents the value of the green color in the guideline.</summary>
+        public const float YellowValue = 0.9f;
+        /// <summary>Represents the value of the transparent color in the guideline. It is a hidden game feature, where invisible guidelines may be created.</summary>
+        public const float GreenValue = 1f;
```
Comment (Altenhh) [inline GDAPI/GDAPI/Utilities/Objects/GeometryDash/General/GuidelineColor.cs]: where the fuCK IS WHITE 🔫

Comment (AlFasGD) [reply GDAPI/GDAPI/Utilities/Objects/GeometryDash/General/GuidelineColor.cs]: There's NO WHITE GUIDELINES

**Comment by AlFasGD** [comment]
What audio jungle stuff? Should this also come with automatic BPM detection and shit?

**Comment by Altenhh** [comment]
too much work, wont be worth it for now

**Comment by AlFasGD** [comment]
For another day™

**Pull request merged**